draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -4822,16 +4785,16 @@ Proxy-Authorization: basic aGVsbG86d29ybGQ=
    of a server, without implying a resource action.
 </t>
 <t>
-   An OPTIONS request with an asterisk ("*") as the request-target
-   (<xref target="request.target"/>) applies to the server in general rather than to a
+   An OPTIONS request with an asterisk ("*") as the target URI's path component
```

Comment:
No, this change is wrong. It is not a URI and not a path component.

Reply:
We still need to differentiate between request-target and target URI. They are not the same thing. Likewise for all the below cases of "*" and CONNECT.